presto-main/src/main/java/io/prestosql/server/InternalAuthenticationManager.java

```diff
@@ -76,6 +76,12 @@ public InternalAuthenticationManager(String sharedSecret, String nodeId)
         this.nodeId = nodeId;
     }
 
+    /**
+     * Checks if a request was made by an internal user, looking if was defined some value for X-Presto-Bearer header.
```

Comment:
I will add something like this sentence on the description and not on the summary: 
*Looks if any value for X-Presto-Bearer header was defined. If so, ....*

Reply:
done